docs/sources/setup-grafana/installation/suse-opensuse/index.md

```diff
@@ -0,0 +1,179 @@
+---
+description: Install guide for Grafana on SUSE or OpenSUSE.
+title: Install Grafana on SUSE or OpenSUSE
+menuTitle: Redhat, RHEL, or Fedora
+weight: 450
+---
+
+# Install Grafana on SUSE or OpenSUSE
```

Comment:
@DanCech , I believe I'm correct that the instructions for SUSE/OpenSUSE are exactly the same as the Red hat etc instructions. Is that the case?

Reply:
Ok, it looks like you've added those suggestions, which I accepted.